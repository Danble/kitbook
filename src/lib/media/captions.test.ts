import { parseCaptionsXml, parseTracksXml } from './captions';
// import fetch from 'isomorphic-unfetch';
// improve fetch mock with https://kentcdodds.com/blog/stop-mocking-fetch

// test('getTracks should get tracks', async () => {
//   global.fetch = fetch;
//   expect(await getTracks('rokGy0huYEA')).toMatchInlineSnapshot();
// });

test('parseTracksXml returns array even when there is 1 track', () => {
  expect(parseTracksXml('<?xml version="1.0" encoding="utf-8" ?><transcript_list docid="14931616755950075552"><track id="0" name="" lang_code="en" lang_original="English" lang_translated="English" lang_default="true"/></transcript_list>')).toHaveLength(1);
});

test('parseTracksXml returns array of tracks', () => {
  expect(
    parseTracksXml(`<?xml version="1.0" encoding="utf-8" ?><transcript_list docid="12576590904572141632"><track id="0" name="" lang_code="ar" lang_original="العربية" lang_translated="Arabic"/><track id="2" name="" lang_code="bn" lang_original="বাংলা" lang_translated="Bangla"/><track id="1" name="" lang_code="bg" lang_original="Български" lang_translated="Bulgarian"/><track id="3" name="" lang_code="ca" lang_original="Català" lang_translated="Catalan"/><track id="54" name="" lang_code="zh-HK" lang_original="中文（香港）" lang_translated="Chinese (Hong Kong)"/><track id="53" name="" lang_code="zh-CN" lang_original="中文（简体）" lang_translated="Chinese (Simplified)"/><track id="55" name="" lang_code="zh-TW" lang_original="中文（繁體）" lang_translated="Chinese (Traditional)"/><track id="21" name="" lang_code="hr" lang_original="Hrvatski" lang_translated="Croatian"/><track 
    id="4" name="" lang_code="cs" lang_original="Čeština" lang_translated="Czech"/><track id="5" name="" lang_code="da" lang_original="Dansk" lang_translated="Danish"/><track id="33" name="" lang_code="nl-BE" lang_original="Nederlands (België)" lang_translated="Dutch (Belgium)"/><track id="34" name="" lang_code="nl-NL" lang_original="Nederlands (Nederland)" lang_translated="Dutch (Netherlands)"/><track id="8" name="" lang_code="en-GB" lang_original="English (United Kingdom)" lang_translated="English (United Kingdom)" lang_default="true"/><track id="9" name="CC" lang_code="en-US" lang_original="English (United States)" lang_translated="English (United States)"/><track id="12" name="" lang_code="et" lang_original="Eesti" lang_translated="Estonian"/><track id="15" name="" lang_code="fil" lang_original="Filipino" lang_translated="Filipino"/><track id="14" name="" 
    lang_code="fi" lang_original="Suomi" lang_translated="Finnish"/><track id="16" name="" lang_code="fr-BE" lang_original="Français (Belgique)" lang_translated="French (Belgium)"/><track id="17" name="" lang_code="fr-CA" lang_original="Français (Canada)" lang_translated="French (Canada)"/><track id="18" name="" lang_code="fr-FR" lang_original="Français (France)" lang_translated="French (France)"/><track id="6" name="" lang_code="de" 
    lang_original="Deutsch" lang_translated="German"/><track id="7" name="" lang_code="el" lang_original="Ελληνικά" lang_translated="Greek"/><track id="19" name="" lang_code="gu" lang_original="ગુજરાતી" lang_translated="G
    ujarati"/><track id="24" name="" lang_code="iw" lang_original="עברית" lang_translated="Hebrew"/><track id="20" name="" lang_code="hi" lang_original="हिन्दी" lang_translated="Hindi"/><track id="22" name="" lang_code="i
    d" lang_original="Indonesia" lang_translated="Indonesian"/><track id="23" name="" lang_code="it" lang_original="Italiano" lang_translated="Italian"/><track id="25" name="" lang_code="ja" lang_original="日本語" lang_translated="Japanese"/><track id="26" name="" lang_code="kn" lang_original="ಕನ್ನಡ" lang_translated="Kannada"/><track id="27" name="" lang_code="ko" lang_original="한국어" lang_translated="Korean"/><track id="29" name=""
     lang_code="lv" lang_original="Latviešu" lang_translated="Latvian"/><track id="28" name="" lang_code="lt" lang_original="Lietuvių" lang_translated="Lithuanian"/><track id="32" name="" lang_code="ms" lang_original="Melayu" lang_translated="Malay"/><track id="30" name="" lang_code="ml" lang_original="മലയാളം" lang_translated="Malayalam"/><track id="31" name="" lang_code="mr" lang_original="मराठी" lang_translated="Marathi"/><track id="35" name="" lang_code="no" lang_original="Norsk" lang_translated="Norwegian"/><track id="13" name="" lang_code="fa" lang_original="فارسی" lang_translated="Persian"/><track id="37" name="" lang_code="pl" lang_original="Polski" lang_translated="Polish"/><track id="38" name="" lang_code="pt-BR" lang_original="Português (Brasil)" lang_translated="Portuguese (Brazil)"/><track id="39" name="" lang_code="pt-PT" lang_original="Português 
    (Portugal)" lang_translated="Portuguese (Portugal)"/><track id="36" name="" lang_code="pa" lang_original="ਪੰਜਾਬੀ" lang_translated="Punjabi"/><track id="40" name="" lang_code="ro" lang_original="Română" lang_translated
    ="Romanian"/><track id="41" name="" lang_code="ru" lang_original="Русский" lang_translated="Russian"/><track id="44" name="" lang_code="sr" lang_original="Српски" lang_translated="Serbian"/><track id="42" name="" lang_code="sk" lang_original="Slovenčina" lang_translated="Slovak"/><track id="43" name="" lang_code="sl" lang_original="Slovenščina" lang_translated="Slovenian"/><track id="10" name="" lang_code="es-419" lang_original="Español (Latinoamérica)" lang_translated="Spanish (Latin America)"/><track id="11" name="" lang_code="es-ES" lang_original="Español (España)" lang_translated="Spanish (Spain)"/><track id="46" name="" lang_code="sw" lang_original="Kiswahili" lang_translated="Swahili"/><track id="45" name="" lang_code="sv" lang_original="Svenska" lang_translated="Swedish"/><track id="47" name="" lang_code="ta" lang_original="தமிழ்" lang_translated="T
    amil"/><track id="48" name="" lang_code="te" lang_original="తెలుగు" lang_translated="Telugu"/><track id="49" name="" lang_code="th" lang_original="ไทย" lang_translated="Thai"/><track id="50" name="" lang_code="tr" lan
    g_original="Türkçe" lang_translated="Turkish"/><track id="51" name="" lang_code="uk" lang_original="Українська" lang_translated="Ukrainian"/><track id="52" name="" lang_code="vi" lang_original="Tiếng Việt" lang_translated="Vietnamese"/></transcript_list>`)
  ).toMatchSnapshot();
});

test('parseCaptionsXml', () => {
  expect(
    parseCaptionsXml(`<?xml version="1.0" encoding="utf-8" ?><transcript><text start="2.102" dur="4.122">So Glasgow got a great theatre,
  A lively crowd
  And a great atmosphere</text><text start="6.725" dur="2.6">So far though
  There is one thing missing</text><text start="9.526" dur="0.7">Talent!</text><text start="12.127" dur="5.9">Yes, tensions are high backstage
  As everyone wonders where that
  Talent is going to come from?</text><text start="18.228" dur="5.1">Next up is a contestant who thinks
  She has what it takes
  To put Glasgow on the map</text><text start="24.729" dur="2">Alright
  How are you doing?
  I&amp;#39;m very well</text><text start="26.83" dur="5.5">My name is Susan Boyle
  I&amp;#39;m nearly 48
  Currently unemployed but i&amp;#39;m still looking</text><text start="32.331" dur="2.8">I&amp;#39;m going to sing for you on
  Britains Got Talent today</text><text start="35.232" dur="4">A little nervous?
  Yeah Sure!
  That&amp;#39;s not surprising</text><text start="40.133" dur="3.2">At the moment I live
  On my own with my cat
  Called Pebbles</text><text start="43.434" dur="6.6">i&amp;#39;ve never been married
  Never been kissed
  Oh shame, but its not and advert!</text><text start="50.335" dur="3.1">Have you done this for many a year?
  Since I was twelve</text><text start="54.136" dur="8.4">I have always wanted to
  Perform in front of a large audience
  I&amp;#39;m going to make that audience ROCK!</text><text start="68.137" dur="6.1">Hi, what&amp;#39;s your name darling?
  My name is Susan Boyle
  Ok, Susan where are you from?</text><text start="74.438" dur="2.1">I&amp;#39;m from Blackburn
  Near Bathgate
  West Lothian</text><text start="76.539" dur="4.8">Its a big town?
  its a sort of a collection of
  A collection of eh</text><text start="82.94" dur="3.8">Villages!
  Had to think there
  And how old are you Susan?</text><text start="86.741" dur="2.5">I&amp;#39;m 47</text><text start="90.942" dur="4.3">And that&amp;#39;s just one side of me</text><text start="100.143" dur="5.1">Ok whats the dream?
  I&amp;#39;m trying to be a professional singer</text><text start="105.244" dur="4.3">And why hasn&amp;#39;t it
  Worked out so far Susan?
  I haven&amp;#39;t been giving the chance before</text><text start="109.545" dur="4.6">But heres hoping it will change?
  Who would you like to be
  As successful as?</text><text start="114.346" dur="1.6">Elaine Paige!</text><text start="115.947" dur="1.2">What are you going to
  Sing for us?</text><text start="117.248" dur="3.5">I&amp;#39;m going to sing
  I Dreamed A Dream
  From Les Miserables</text><text start="120.749" dur="6.3">Ok Big Song!
  Yeah
  Yeah!</text><text start="137.35" dur="5.7">I dreamed a dream
  In time gone by
  ( Cheering and Applause )</text><text start="144.051" dur="6">When hope was high
  And life worth living
  ( You didnt expect that did you? No )</text><text start="150.052" dur="6.5">I dreamed that love
  Would never die</text><text start="157.253" dur="5.5">I dreamed that
  God would be forgiving</text><text start="163.454" dur="5.5">Then I was young
  And unafraid</text><text start="169.855" dur="7">And dreams were made
  And used and wasted</text><text start="177.856" dur="5.8">There was no ransom
  to be paid</text><text start="183.757" dur="5.5">No song unsung
  No wine untasted</text><text start="190.358" dur="12.7">But the tigers come at night
  With their voices soft as thunder</text><text start="203.159" dur="19.4">As they tear your hope apart
  And they turn your dream to shame
  ( Look at that! )</text><text start="222.56" dur="12.6">And still I dream he&amp;#39;ll come to me
  That we will live the years together</text><text start="235.361" dur="13.3">But there are dreams that cannot be
  And there are storms we cannot weather</text><text start="251.762" dur="11.1">I had a dream my life would be
  So different from this hell I&amp;#39;m living</text><text start="262.963" dur="15">So different now from what it seemed
  Now life has killed the dream I dreamed.</text><text start="278.064" dur="10">HUGE APPLAUSE !</text><text start="290.565" dur="8.6">Hey, Come back here, wait, wait
  Alright, Thank you very much, Susan
  Ok Piers</text><text start="299.366" dur="7.4">Without a doubt that
  Is the biggest surprise I have had
  In my three years on the show</text><text start="307.767" dur="6.2">When you stood there
  With that cheeky grin
  And said &amp;quot;I want to be like Elaine Paige&amp;quot;</text><text start="313.968" dur="4.9">And everyone was laughing at you
  No-One is laughing now!
  That was STUNNING!</text><text start="318.869" dur="4.5">An incredible perfomance
  (APPLAUSE)</text><text start="324.17" dur="4.1">Amazing!
  I&amp;#39;m reeling from shock
  I don&amp;#39;t know what you thought of it?</text><text start="328.371" dur="4.9">I&amp;#39;m so thrilled because
  I know everybody was against you</text><text start="333.372" dur="7">I honestly think we were all
  Being very cynical and that&amp;#39;s
  The biggest wake-up call ever</text><text start="340.373" dur="7.2">And i just want to say
  It was a complete privilege
  Listening to that (APPLAUSE) (Thank You)</text><text start="348.574" dur="5.4">Susan I knew the minute
  You walk out ( Laughter)
  Oh Simon!</text><text start="354.075" dur="8.4">On that stage that we were
  Going to hear something extraordinary
  And I was right ( Laughter)</text><text start="362.576" dur="1.9">What a load of tosh!</text><text start="364.477" dur="5.3">Susan, you are a little tiger
  Aren&amp;#39;t you?
  I don&amp;#39;t know about that</text><text start="369.778" dur="3.2">You are, Ok, moment of truth
  Piers, Yes or No?</text><text start="372.979" dur="5.4">The biggest YES I have
  Ever given anybody</text><text start="378.88" dur="8.6">Amanda? Yes, definitley
  Susan Boyle, you can go back
  To your village</text><text start="387.781" dur="7.5">With your head held high
  Its three Yeses</text><text start="397.982" dur="8.9">HUGE APPLAUSE
  (Thank you)</text><text start="410.783" dur="4.3">I think you enjoyed
  That just a little!
  You maybe right</text><text start="415.784" dur="3.3">What a voice!
  Incredible</text><text start="419.385" dur="6">Congratulations!
  Oh my god!
  How do you feel?</text><text start="425.486" dur="1.7">Bloody fantastic</text><text start="427.187" dur="6.7">Piers said thats the biggest YES
  He&amp;#39;s ever given on the show
  In three series, Oh my god!</text><text start="433.888" dur="5">Mind blowing
  The most extraordinary shock
  We&amp;#39;ve ever had</text><text start="438.889" dur="7.1">Just so emotional
  Unbelieveable, Emotional
  And Fantastic</text></transcript>`)
  ).toMatchSnapshot();
});
