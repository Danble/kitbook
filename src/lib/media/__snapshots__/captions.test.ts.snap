// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseCaptionsXml 1`] = `
Array [
  Object {
    "duration": 4.122,
    "start": 2.102,
    "text": "So Glasgow got a great theatre,
  A lively crowd
  And a great atmosphere",
  },
  Object {
    "duration": 2.6,
    "start": 6.725,
    "text": "So far though
  There is one thing missing",
  },
  Object {
    "duration": 0.7,
    "start": 9.526,
    "text": "Talent!",
  },
  Object {
    "duration": 5.9,
    "start": 12.127,
    "text": "Yes, tensions are high backstage
  As everyone wonders where that
  Talent is going to come from?",
  },
  Object {
    "duration": 5.1,
    "start": 18.228,
    "text": "Next up is a contestant who thinks
  She has what it takes
  To put Glasgow on the map",
  },
  Object {
    "duration": 2,
    "start": 24.729,
    "text": "Alright
  How are you doing?
  I&#39;m very well",
  },
  Object {
    "duration": 5.5,
    "start": 26.83,
    "text": "My name is Susan Boyle
  I&#39;m nearly 48
  Currently unemployed but i&#39;m still looking",
  },
  Object {
    "duration": 2.8,
    "start": 32.331,
    "text": "I&#39;m going to sing for you on
  Britains Got Talent today",
  },
  Object {
    "duration": 4,
    "start": 35.232,
    "text": "A little nervous?
  Yeah Sure!
  That&#39;s not surprising",
  },
  Object {
    "duration": 3.2,
    "start": 40.133,
    "text": "At the moment I live
  On my own with my cat
  Called Pebbles",
  },
  Object {
    "duration": 6.6,
    "start": 43.434,
    "text": "i&#39;ve never been married
  Never been kissed
  Oh shame, but its not and advert!",
  },
  Object {
    "duration": 3.1,
    "start": 50.335,
    "text": "Have you done this for many a year?
  Since I was twelve",
  },
  Object {
    "duration": 8.4,
    "start": 54.136,
    "text": "I have always wanted to
  Perform in front of a large audience
  I&#39;m going to make that audience ROCK!",
  },
  Object {
    "duration": 6.1,
    "start": 68.137,
    "text": "Hi, what&#39;s your name darling?
  My name is Susan Boyle
  Ok, Susan where are you from?",
  },
  Object {
    "duration": 2.1,
    "start": 74.438,
    "text": "I&#39;m from Blackburn
  Near Bathgate
  West Lothian",
  },
  Object {
    "duration": 4.8,
    "start": 76.539,
    "text": "Its a big town?
  its a sort of a collection of
  A collection of eh",
  },
  Object {
    "duration": 3.8,
    "start": 82.94,
    "text": "Villages!
  Had to think there
  And how old are you Susan?",
  },
  Object {
    "duration": 2.5,
    "start": 86.741,
    "text": "I&#39;m 47",
  },
  Object {
    "duration": 4.3,
    "start": 90.942,
    "text": "And that&#39;s just one side of me",
  },
  Object {
    "duration": 5.1,
    "start": 100.143,
    "text": "Ok whats the dream?
  I&#39;m trying to be a professional singer",
  },
  Object {
    "duration": 4.3,
    "start": 105.244,
    "text": "And why hasn&#39;t it
  Worked out so far Susan?
  I haven&#39;t been giving the chance before",
  },
  Object {
    "duration": 4.6,
    "start": 109.545,
    "text": "But heres hoping it will change?
  Who would you like to be
  As successful as?",
  },
  Object {
    "duration": 1.6,
    "start": 114.346,
    "text": "Elaine Paige!",
  },
  Object {
    "duration": 1.2,
    "start": 115.947,
    "text": "What are you going to
  Sing for us?",
  },
  Object {
    "duration": 3.5,
    "start": 117.248,
    "text": "I&#39;m going to sing
  I Dreamed A Dream
  From Les Miserables",
  },
  Object {
    "duration": 6.3,
    "start": 120.749,
    "text": "Ok Big Song!
  Yeah
  Yeah!",
  },
  Object {
    "duration": 5.7,
    "start": 137.35,
    "text": "I dreamed a dream
  In time gone by
  ( Cheering and Applause )",
  },
  Object {
    "duration": 6,
    "start": 144.051,
    "text": "When hope was high
  And life worth living
  ( You didnt expect that did you? No )",
  },
  Object {
    "duration": 6.5,
    "start": 150.052,
    "text": "I dreamed that love
  Would never die",
  },
  Object {
    "duration": 5.5,
    "start": 157.253,
    "text": "I dreamed that
  God would be forgiving",
  },
  Object {
    "duration": 5.5,
    "start": 163.454,
    "text": "Then I was young
  And unafraid",
  },
  Object {
    "duration": 7,
    "start": 169.855,
    "text": "And dreams were made
  And used and wasted",
  },
  Object {
    "duration": 5.8,
    "start": 177.856,
    "text": "There was no ransom
  to be paid",
  },
  Object {
    "duration": 5.5,
    "start": 183.757,
    "text": "No song unsung
  No wine untasted",
  },
  Object {
    "duration": 12.7,
    "start": 190.358,
    "text": "But the tigers come at night
  With their voices soft as thunder",
  },
  Object {
    "duration": 19.4,
    "start": 203.159,
    "text": "As they tear your hope apart
  And they turn your dream to shame
  ( Look at that! )",
  },
  Object {
    "duration": 12.6,
    "start": 222.56,
    "text": "And still I dream he&#39;ll come to me
  That we will live the years together",
  },
  Object {
    "duration": 13.3,
    "start": 235.361,
    "text": "But there are dreams that cannot be
  And there are storms we cannot weather",
  },
  Object {
    "duration": 11.1,
    "start": 251.762,
    "text": "I had a dream my life would be
  So different from this hell I&#39;m living",
  },
  Object {
    "duration": 15,
    "start": 262.963,
    "text": "So different now from what it seemed
  Now life has killed the dream I dreamed.",
  },
  Object {
    "duration": 10,
    "start": 278.064,
    "text": "HUGE APPLAUSE !",
  },
  Object {
    "duration": 8.6,
    "start": 290.565,
    "text": "Hey, Come back here, wait, wait
  Alright, Thank you very much, Susan
  Ok Piers",
  },
  Object {
    "duration": 7.4,
    "start": 299.366,
    "text": "Without a doubt that
  Is the biggest surprise I have had
  In my three years on the show",
  },
  Object {
    "duration": 6.2,
    "start": 307.767,
    "text": "When you stood there
  With that cheeky grin
  And said &quot;I want to be like Elaine Paige&quot;",
  },
  Object {
    "duration": 4.9,
    "start": 313.968,
    "text": "And everyone was laughing at you
  No-One is laughing now!
  That was STUNNING!",
  },
  Object {
    "duration": 4.5,
    "start": 318.869,
    "text": "An incredible perfomance
  (APPLAUSE)",
  },
  Object {
    "duration": 4.1,
    "start": 324.17,
    "text": "Amazing!
  I&#39;m reeling from shock
  I don&#39;t know what you thought of it?",
  },
  Object {
    "duration": 4.9,
    "start": 328.371,
    "text": "I&#39;m so thrilled because
  I know everybody was against you",
  },
  Object {
    "duration": 7,
    "start": 333.372,
    "text": "I honestly think we were all
  Being very cynical and that&#39;s
  The biggest wake-up call ever",
  },
  Object {
    "duration": 7.2,
    "start": 340.373,
    "text": "And i just want to say
  It was a complete privilege
  Listening to that (APPLAUSE) (Thank You)",
  },
  Object {
    "duration": 5.4,
    "start": 348.574,
    "text": "Susan I knew the minute
  You walk out ( Laughter)
  Oh Simon!",
  },
  Object {
    "duration": 8.4,
    "start": 354.075,
    "text": "On that stage that we were
  Going to hear something extraordinary
  And I was right ( Laughter)",
  },
  Object {
    "duration": 1.9,
    "start": 362.576,
    "text": "What a load of tosh!",
  },
  Object {
    "duration": 5.3,
    "start": 364.477,
    "text": "Susan, you are a little tiger
  Aren&#39;t you?
  I don&#39;t know about that",
  },
  Object {
    "duration": 3.2,
    "start": 369.778,
    "text": "You are, Ok, moment of truth
  Piers, Yes or No?",
  },
  Object {
    "duration": 5.4,
    "start": 372.979,
    "text": "The biggest YES I have
  Ever given anybody",
  },
  Object {
    "duration": 8.6,
    "start": 378.88,
    "text": "Amanda? Yes, definitley
  Susan Boyle, you can go back
  To your village",
  },
  Object {
    "duration": 7.5,
    "start": 387.781,
    "text": "With your head held high
  Its three Yeses",
  },
  Object {
    "duration": 8.9,
    "start": 397.982,
    "text": "HUGE APPLAUSE
  (Thank you)",
  },
  Object {
    "duration": 4.3,
    "start": 410.783,
    "text": "I think you enjoyed
  That just a little!
  You maybe right",
  },
  Object {
    "duration": 3.3,
    "start": 415.784,
    "text": "What a voice!
  Incredible",
  },
  Object {
    "duration": 6,
    "start": 419.385,
    "text": "Congratulations!
  Oh my god!
  How do you feel?",
  },
  Object {
    "duration": 1.7,
    "start": 425.486,
    "text": "Bloody fantastic",
  },
  Object {
    "duration": 6.7,
    "start": 427.187,
    "text": "Piers said thats the biggest YES
  He&#39;s ever given on the show
  In three series, Oh my god!",
  },
  Object {
    "duration": 5,
    "start": 433.888,
    "text": "Mind blowing
  The most extraordinary shock
  We&#39;ve ever had",
  },
  Object {
    "duration": 7.1,
    "start": 438.889,
    "text": "Just so emotional
  Unbelieveable, Emotional
  And Fantastic",
  },
]
`;

exports[`parseTracksXml returns array of tracks 1`] = `
Array [
  Object {
    "id": 0,
    "langCode": "ar",
    "langDefault": false,
    "langOriginal": "العربية",
    "langTranslated": "Arabic",
    "name": "",
  },
  Object {
    "id": 2,
    "langCode": "bn",
    "langDefault": false,
    "langOriginal": "বাংলা",
    "langTranslated": "Bangla",
    "name": "",
  },
  Object {
    "id": 1,
    "langCode": "bg",
    "langDefault": false,
    "langOriginal": "Български",
    "langTranslated": "Bulgarian",
    "name": "",
  },
  Object {
    "id": 3,
    "langCode": "ca",
    "langDefault": false,
    "langOriginal": "Català",
    "langTranslated": "Catalan",
    "name": "",
  },
  Object {
    "id": 54,
    "langCode": "zh-HK",
    "langDefault": false,
    "langOriginal": "中文（香港）",
    "langTranslated": "Chinese (Hong Kong)",
    "name": "",
  },
  Object {
    "id": 53,
    "langCode": "zh-CN",
    "langDefault": false,
    "langOriginal": "中文（简体）",
    "langTranslated": "Chinese (Simplified)",
    "name": "",
  },
  Object {
    "id": 55,
    "langCode": "zh-TW",
    "langDefault": false,
    "langOriginal": "中文（繁體）",
    "langTranslated": "Chinese (Traditional)",
    "name": "",
  },
  Object {
    "id": 21,
    "langCode": "hr",
    "langDefault": false,
    "langOriginal": "Hrvatski",
    "langTranslated": "Croatian",
    "name": "",
  },
  Object {
    "id": 4,
    "langCode": "cs",
    "langDefault": false,
    "langOriginal": "Čeština",
    "langTranslated": "Czech",
    "name": "",
  },
  Object {
    "id": 5,
    "langCode": "da",
    "langDefault": false,
    "langOriginal": "Dansk",
    "langTranslated": "Danish",
    "name": "",
  },
  Object {
    "id": 33,
    "langCode": "nl-BE",
    "langDefault": false,
    "langOriginal": "Nederlands (België)",
    "langTranslated": "Dutch (Belgium)",
    "name": "",
  },
  Object {
    "id": 34,
    "langCode": "nl-NL",
    "langDefault": false,
    "langOriginal": "Nederlands (Nederland)",
    "langTranslated": "Dutch (Netherlands)",
    "name": "",
  },
  Object {
    "id": 8,
    "langCode": "en-GB",
    "langDefault": true,
    "langOriginal": "English (United Kingdom)",
    "langTranslated": "English (United Kingdom)",
    "name": "",
  },
  Object {
    "id": 9,
    "langCode": "en-US",
    "langDefault": false,
    "langOriginal": "English (United States)",
    "langTranslated": "English (United States)",
    "name": "CC",
  },
  Object {
    "id": 12,
    "langCode": "et",
    "langDefault": false,
    "langOriginal": "Eesti",
    "langTranslated": "Estonian",
    "name": "",
  },
  Object {
    "id": 15,
    "langCode": "fil",
    "langDefault": false,
    "langOriginal": "Filipino",
    "langTranslated": "Filipino",
    "name": "",
  },
  Object {
    "id": 14,
    "langCode": "fi",
    "langDefault": false,
    "langOriginal": "Suomi",
    "langTranslated": "Finnish",
    "name": "",
  },
  Object {
    "id": 16,
    "langCode": "fr-BE",
    "langDefault": false,
    "langOriginal": "Français (Belgique)",
    "langTranslated": "French (Belgium)",
    "name": "",
  },
  Object {
    "id": 17,
    "langCode": "fr-CA",
    "langDefault": false,
    "langOriginal": "Français (Canada)",
    "langTranslated": "French (Canada)",
    "name": "",
  },
  Object {
    "id": 18,
    "langCode": "fr-FR",
    "langDefault": false,
    "langOriginal": "Français (France)",
    "langTranslated": "French (France)",
    "name": "",
  },
  Object {
    "id": 6,
    "langCode": "de",
    "langDefault": false,
    "langOriginal": "Deutsch",
    "langTranslated": "German",
    "name": "",
  },
  Object {
    "id": 7,
    "langCode": "el",
    "langDefault": false,
    "langOriginal": "Ελληνικά",
    "langTranslated": "Greek",
    "name": "",
  },
  Object {
    "id": 19,
    "langCode": "gu",
    "langDefault": false,
    "langOriginal": "ગુજરાતી",
    "langTranslated": "G     ujarati",
    "name": "",
  },
  Object {
    "id": 24,
    "langCode": "iw",
    "langDefault": false,
    "langOriginal": "עברית",
    "langTranslated": "Hebrew",
    "name": "",
  },
  Object {
    "id": 20,
    "langCode": "hi",
    "langDefault": false,
    "langOriginal": "हिन्दी",
    "langTranslated": "Hindi",
    "name": "",
  },
  Object {
    "id": 22,
    "langCode": "i     d",
    "langDefault": false,
    "langOriginal": "Indonesia",
    "langTranslated": "Indonesian",
    "name": "",
  },
  Object {
    "id": 23,
    "langCode": "it",
    "langDefault": false,
    "langOriginal": "Italiano",
    "langTranslated": "Italian",
    "name": "",
  },
  Object {
    "id": 25,
    "langCode": "ja",
    "langDefault": false,
    "langOriginal": "日本語",
    "langTranslated": "Japanese",
    "name": "",
  },
  Object {
    "id": 26,
    "langCode": "kn",
    "langDefault": false,
    "langOriginal": "ಕನ್ನಡ",
    "langTranslated": "Kannada",
    "name": "",
  },
  Object {
    "id": 27,
    "langCode": "ko",
    "langDefault": false,
    "langOriginal": "한국어",
    "langTranslated": "Korean",
    "name": "",
  },
  Object {
    "id": 29,
    "langCode": "lv",
    "langDefault": false,
    "langOriginal": "Latviešu",
    "langTranslated": "Latvian",
    "name": "",
  },
  Object {
    "id": 28,
    "langCode": "lt",
    "langDefault": false,
    "langOriginal": "Lietuvių",
    "langTranslated": "Lithuanian",
    "name": "",
  },
  Object {
    "id": 32,
    "langCode": "ms",
    "langDefault": false,
    "langOriginal": "Melayu",
    "langTranslated": "Malay",
    "name": "",
  },
  Object {
    "id": 30,
    "langCode": "ml",
    "langDefault": false,
    "langOriginal": "മലയാളം",
    "langTranslated": "Malayalam",
    "name": "",
  },
  Object {
    "id": 31,
    "langCode": "mr",
    "langDefault": false,
    "langOriginal": "मराठी",
    "langTranslated": "Marathi",
    "name": "",
  },
  Object {
    "id": 35,
    "langCode": "no",
    "langDefault": false,
    "langOriginal": "Norsk",
    "langTranslated": "Norwegian",
    "name": "",
  },
  Object {
    "id": 13,
    "langCode": "fa",
    "langDefault": false,
    "langOriginal": "فارسی",
    "langTranslated": "Persian",
    "name": "",
  },
  Object {
    "id": 37,
    "langCode": "pl",
    "langDefault": false,
    "langOriginal": "Polski",
    "langTranslated": "Polish",
    "name": "",
  },
  Object {
    "id": 38,
    "langCode": "pt-BR",
    "langDefault": false,
    "langOriginal": "Português (Brasil)",
    "langTranslated": "Portuguese (Brazil)",
    "name": "",
  },
  Object {
    "id": 39,
    "langCode": "pt-PT",
    "langDefault": false,
    "langOriginal": "Português      (Portugal)",
    "langTranslated": "Portuguese (Portugal)",
    "name": "",
  },
  Object {
    "id": 36,
    "langCode": "pa",
    "langDefault": false,
    "langOriginal": "ਪੰਜਾਬੀ",
    "langTranslated": "Punjabi",
    "name": "",
  },
  Object {
    "id": 40,
    "langCode": "ro",
    "langDefault": false,
    "langOriginal": "Română",
    "langTranslated": "Romanian",
    "name": "",
  },
  Object {
    "id": 41,
    "langCode": "ru",
    "langDefault": false,
    "langOriginal": "Русский",
    "langTranslated": "Russian",
    "name": "",
  },
  Object {
    "id": 44,
    "langCode": "sr",
    "langDefault": false,
    "langOriginal": "Српски",
    "langTranslated": "Serbian",
    "name": "",
  },
  Object {
    "id": 42,
    "langCode": "sk",
    "langDefault": false,
    "langOriginal": "Slovenčina",
    "langTranslated": "Slovak",
    "name": "",
  },
  Object {
    "id": 43,
    "langCode": "sl",
    "langDefault": false,
    "langOriginal": "Slovenščina",
    "langTranslated": "Slovenian",
    "name": "",
  },
  Object {
    "id": 10,
    "langCode": "es-419",
    "langDefault": false,
    "langOriginal": "Español (Latinoamérica)",
    "langTranslated": "Spanish (Latin America)",
    "name": "",
  },
  Object {
    "id": 11,
    "langCode": "es-ES",
    "langDefault": false,
    "langOriginal": "Español (España)",
    "langTranslated": "Spanish (Spain)",
    "name": "",
  },
  Object {
    "id": 46,
    "langCode": "sw",
    "langDefault": false,
    "langOriginal": "Kiswahili",
    "langTranslated": "Swahili",
    "name": "",
  },
  Object {
    "id": 45,
    "langCode": "sv",
    "langDefault": false,
    "langOriginal": "Svenska",
    "langTranslated": "Swedish",
    "name": "",
  },
  Object {
    "id": 47,
    "langCode": "ta",
    "langDefault": false,
    "langOriginal": "தமிழ்",
    "langTranslated": "T     amil",
    "name": "",
  },
  Object {
    "id": 48,
    "langCode": "te",
    "langDefault": false,
    "langOriginal": "తెలుగు",
    "langTranslated": "Telugu",
    "name": "",
  },
  Object {
    "id": 49,
    "langCode": "th",
    "langDefault": false,
    "langOriginal": "ไทย",
    "langTranslated": "Thai",
    "name": "",
  },
  Object {
    "id": 50,
    "langCode": "tr",
    "langDefault": false,
    "langOriginal": undefined,
    "langTranslated": "Turkish",
    "name": "",
  },
  Object {
    "id": 51,
    "langCode": "uk",
    "langDefault": false,
    "langOriginal": "Українська",
    "langTranslated": "Ukrainian",
    "name": "",
  },
  Object {
    "id": 52,
    "langCode": "vi",
    "langDefault": false,
    "langOriginal": "Tiếng Việt",
    "langTranslated": "Vietnamese",
    "name": "",
  },
]
`;
